ingestion/functions/retrieval/retrieval_test.py

```diff
@@ -373,3 +376,24 @@ def test_upload_to_s3_raises_error_on_s3_error(
 
     # We got the wrong exception or no exception, fail the test.
     assert not "Should have raised an exception."
+
+def test_decompress_zip_file():
```

Comment:
nit, but `test_raw_content_unzips` and `test_raw_content_ignores_unknown_mimetypes`?

Reply:
done